### DISCUSSION TITLE
2005/07/21/%E6%88%91%E5%8F%AF%E6%80%9C%E7%9A%84%E7%89%99

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#41] [Announcements]
# 2005/07/21/%E6%88%91%E5%8F%AF%E6%80%9C%E7%9A%84%E7%89%99

医生在我的牙上发现两个大洞，而且都已经很深了。

https://blog.qizhen.xyz/2005/07/21/%E6%88%91%E5%8F%AF%E6%80%9C%E7%9A%84%E7%89%99

**Comment by ruanqizhen** [comment]
yoyo
2005年07月28日 21:17 编辑
我也有颗蛀牙，忍了很久，快不行了，上次补牙偷懒没弄，只能再找个时间补了。

回复
